doc/dependency_decisions.yml

```
@@ -0,0 +1,207 @@
+---
+################################################################################
+#
+# IMPORTANT: This file includes some important legal decisions that were
+# made by the legal team so before making changes, reach out to the slack
+# channel #licenses-tech-stack to ask around and get more context.
+#
+################################################################################
+
+########################################
+#
+# LICENSES WE CAN USE WITHOUT CONCERN
+#
+########################################
+
+- [permit, 2-clause BSDL]
+- [permit, Apache 2.0]
+- [permit, Apache License (2.0)]
+- [permit, BlueOak-1.0.0]
+- [permit, BSD Zero Clause License]
+- [permit, BSD]
+- [permit, Hippocratic-2.1]
+- [permit, ISC]
+- [permit, MIT or MPL-2.0]
+- [permit, MIT]
+- [permit, MIT-0]
+- [permit, New BSD]
+- [permit, Public Domain]
+- [permit, Python-2.0]
+- [permit, ruby]
+- [permit, Simplified BSD]
+- [permit, Unlicense]
+- [permit, WTFPL]
+- [permit, Zlib]
+- [permit, CC0 1.0 Universal]
+- - :approve
+  - Pennylane
+  - :why: Our own software
+- - :approve
+  - pennylane
+  - :why: Our own software
```

why putting it twice?
```suggestion
- - :approve
  - Pennylane
  - :why: Our own software
- - :approve
  - pennylane
  - :why: Our own software
```

---

ah case sensitive?
I'm not sure I get where this is used